### PR TITLE
Use double splats arguements for ruby 3

### PR DESCRIPTION
**Pull request by longnt80**
Need to update double splats argument to ruby 3 syntax

### CODE REVIEW

File: lib/envied.rb [hunk 1]
```diff
@@ -15,13 +15,13 @@ class << self
 
   def self.require(*args, **options)
     requested_groups = (args && !args.empty?) ? args : ENV['ENVIED_GROUPS']
-    env!(requested_groups, options)
+    env!(requested_groups, **options)
     error_on_duplicate_variables!(options)
```
Comment (andrewcameronsims) [inline lib/envied.rb:19]: Is there no need to double splat this one too?

**Review by andrewcameronsims — APPROVED** [review]
Looks good, assuming that you've tried this out as a dependency for meander and there are no problems there.

File: lib/envied.rb (hunk 1 above)
Comment (longnt80) [reply lib/envied.rb:19]: This one uses a normal argument

**Comment by longnt80** [comment]
> Looks good, assuming that you've tried this out as a dependency for meander and there are no problems there.

@andrewcameronsims I've tested using the source from the gem folder for meander
I'll test one more time with the updated repo

**Review by michebble — COMMENTED** [review]
LGTM

**Pull request merged**